### PR TITLE
Playground will now render the returned scene from createScene() when there are multiple scenes added to engine

### PR DESCRIPTION
**Pull request by belfortk**
Fix for #[7295](https://github.com/BabylonJS/Babylon.js/issues/7295). 

update the scene used in engine.runRenderLoop callback from engine.scenes[0] to the one returned from createScene() in the playground script.

### CODE REVIEW

**Comment by belfortk** [comment]
I feel like my implementation is not very DRY.

I tried to make it cleaner in this [commit](https://github.com/belfortk/Babylon.js/commit/d72365a7df26217fd89bef0c5372a4be42a22d80#diff-0f38524fed9cda05e0565578fd1ea1c6R201) but it fails to render the default scene script. Any pointers/ideas would be welcomed! =)

File: Playground/js/main.js [hunk 1]
```diff
@@ -138,23 +138,43 @@ compileAndRun = function(parent, fpsLabel) {
...
-                var scene = engine.scenes[0];
-
-                if (scene.activeCamera || scene.activeCameras.length > 0) {
-                    scene.render();
-                }
-
-                fpsLabel.innerHTML = engine.getFps().toFixed() + " fps";
-            }.bind(this));
+            var sceneToRender;
+            if (scene.then) {
```
Comment (sebavan) [inline Playground/js/main.js]: No need to dupplicate so we ll check tomorrow together :-)

Comment (deltakosh) [reply Playground/js/main.js]: Well this is really important to consider that your code is public and must be clean and up to great standards.

**Comment by belfortk** [comment]
Made a smarter PR [here](https://github.com/BabylonJS/Babylon.js/pull/7318)

**Pull request closed without merging**